src/components/UpcomingEvents/style.module.css

```diff
@@ -0,0 +1,124 @@
+.event_page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.events {
+    margin: 0 auto;
+    font-family: Arial, sans-serif;
+    text-align: left;
+  }
+  
+  .event_card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: 20px;
+    padding: 10px;
+    border: 1px solid #ccc;
+    border-radius: 10px;
+    background-color: #f9f9f9;
+    height: 30rem;
+    width: 42rem;
```

Comment:
I don't recommend setting the width and height like this since it doesn't help with the responsiveness